packages/assistify-ai/server/lib/SmartiAdapter.js

```diff
@@ -129,6 +129,11 @@ export class SmartiAdapter {
 			// we at least cannot consider the sync successful
 			Meteor.defer(() => SmartiAdapter._markRoomAsUnsynced(message.rid));
 		}
+
+		// TODO: Setting
```

Comment:
I don't understand what is to be done here. A setting exists at least.

Reply:
forgot to delete the comment after adding the setting ;)